crfs.go

```diff
@@ -946,7 +946,7 @@ func (n *node) Attr(ctx context.Context, a *fuse.Attr) error {
 	a.Uid = uint32(n.te.Uid)
 	a.Gid = uint32(n.te.Gid)
 	a.Rdev = uint32(unix.Mkdev(uint32(n.te.DevMajor), uint32(n.te.DevMinor)))
-	a.Nlink = 1 // TODO: get this from te once hardlinks are more supported
+	a.Nlink = uint32(n.te.Nlink)
```

Comment:
Should this map 0 to 1?

Reply:
Yes, now NumLink=0 means one name references the entry so I fixed it.